erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -365,24 +365,20 @@ def make_material_request(self):
 			key = '{}:{}:{}'.format(item.sales_order, material_request_type, item_doc.customer or '')
 			schedule_date = add_days(nowdate(), cint(item_doc.lead_time_days))
 
-			if not key in material_request_map:
```

Comment:
Why was this `if condition` check removed ?

Reply:
It was trying to add item in the same document and throwing error Submitted Document cannot be changed.